pkg/doc/jsonld/context/importer.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
+
+import (
+	"bufio"
+	"embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/piprate/json-gold/ld"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/spi/storage"
+)
+
+const (
+	DefaultContextDBName = "ContextsDB"
+)
+
+var logger = log.New("aries-framework/jsonld/context")
+
+// Importer imports JSON-LD context documents into the underlying storage.
+type Importer struct {
+	store storage.Store
+}
+
+type importerOpts struct {
+	contextDBName   string
+	preloadContexts bool
+}
+
+// ImporterOpts configures Importer during creation.
+type ImporterOpts func(opts *importerOpts)
+
+// NewImporter returns a new Importer instance.
+func NewImporter(storageProvider storage.Provider, opts ...ImporterOpts) (*Importer, error) {
+	importerOpts := &importerOpts{
+		contextDBName: DefaultContextDBName,
+	}
+
+	for i := range opts {
+		opts[i](importerOpts)
+	}
+
+	store, err := storageProvider.OpenStore(importerOpts.contextDBName)
+	if err != nil {
+		return nil, fmt.Errorf("open store for context DB: %w", err)
+	}
+
+	if importerOpts.preloadContexts {
```

Comment:
Suggest adding some comments that these preloaded contexts are embedded in the compilation.